### PR TITLE
feat: added hover effect to social SVG icons

### PR DESCRIPTION
**Pull request by karthiknadar1204**
## Changes proposed
Added hover effect to the social icons on the navbar, The site currently uses svg for the social icons so had to add the colour as bg, still works perfectly fine.

## Screenshots

Screenshot 2023-06-29 at 9 18 58 AM

### CODE REVIEW

**Review by Susmita-Dey — CHANGES_REQUESTED** [review]
Remove svgs and use react-icons

**Comment by karthiknadar1204** [comment]
@Susmita-Dey I am unable to install react icons, i tried using npm i react-icons but it is showing error message because it is a nextjs based project, any other way to install it?

**Comment by Susmita-Dey** [comment]
> @Susmita-Dey I am unable to install react icons, i tried using npm i react-icons but it is showing error message because it is a nextjs based project, any other way to install it?

There should be any issue. Do `npm i` once and check if it's already there or not in the `package.json` file. Follow this [guide](https://react-icons.github.io/react-icons)

**Comment by karthiknadar1204** [comment]
on it, thank you

**Comment by karthiknadar1204** [comment]
@Susmita-Dey 
Screenshot 2023-06-29 at 10 22 12 AM

This is the error that occurs on running the command

**Comment by karthiknadar1204** [comment]
pnpm add react-icons -w


I am trying out this command for once, will see

**Comment by karthiknadar1204** [comment]
@Susmita-Dey Done

**Comment by karthiknadar1204** [comment]
pushing the changes in a bit

**Comment by karthiknadar1204** [comment]
@Susmita-Dey Done, please check now

File: src/components/HeaderSocialMedia.jsx [hunk 1]
```diff
@@ -1,25 +1,66 @@
...
+        <Link
+          href="https://twitter.com/devs_in_tech"
+          aria-label="Visit us on Twitter"
+          title="Twitter (External Link)"
+          rel="noopener noreferrer"
+          target="_blank"
+        >
+          <FaGithub
+            size={30}
+            className="dark:fill-white hover:fill-[#211F1F] transition-all duration-200 ease-in-out"
```
Comment (Susmita-Dey) [inline src/components/HeaderSocialMedia.jsx:31]: Change it as it's kinda becoming invisible on hover. Also, the text on hover showing me "Twitter (External link)" which is wrong

**Review by Susmita-Dey — CHANGES_REQUESTED** [review]

**Comment by karthiknadar1204** [comment]
ah, in a min

**Comment by karthiknadar1204** [comment]
So hat should i do for hover on the github icon, like any other colour code you have in mind?

**Comment by karthiknadar1204** [comment]
Screenshot 2023-06-29 at 11 27 56 AM

@Susmita-Dey How does this look for github hover

File: src/components/HeaderSocialMedia.jsx (hunk 1 above)
Comment (karthiknadar1204) [reply src/components/HeaderSocialMedia.jsx]: Have pushed the changes implementing the stated changes, please check it now @Susmita-Dey

**Review by Susmita-Dey — APPROVED** [review]
LGTM.

**Review by Susmita-Dey — CHANGES_REQUESTED** [review]
Why did you remove the scale feature on hover?

Fix it

**Comment by karthiknadar1204** [comment]
Will push the changes right away, I think that should be it, five me like 5 mins

**Comment by karthiknadar1204** [comment]
@Susmita-Dey Surely done , I was experimenting with the Links, just amidst all that, I forgot to add back the scale effect

**Review by Susmita-Dey — APPROVED** [review]
LGTM

**Pull request merged**